### DISCUSSION TITLE
Playback settings' expand button shows the wrong state when opening a replay

### DISCUSSION THREAD

**Discussion by Gabixel** [ppy/osu#16343] [Q&A] [answered]
When you watch a replay (or the autoplay) you can notice in the **playback** settings that the button to expand/contract the area is shown in the wrong state. Re-expanding the area will show the button in the correct state.

To show the difference, I opened the visual settings:

![image](https://user-images.githubusercontent.com/43073074/148410124-451ab327-a53b-453e-9522-be1f2ccdf433.png)

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/16344
